cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -338,6 +327,23 @@
             display: none !important;
         }
     } );
+
+    // Hide fallback by default.
+    &_fallback {
+        display: none;
+        padding: 8px;
+        padding-right: 0;
+
+        // Only show fallback if JavaScript (JS) is not enabled.
+        .no-js & {
+            display: block;
+        }
+    }
+}
+
+// Only show global search if JavaScript (JS) is enabled.
+html.js .m-global-search_trigger {
+    display: block;
 }
```

Comment:
I don't love moving this so far from the original `display: hidden`. DId you just to that to have it close to the fallback's styling?

Reply:
I forget why I put it at the bottom. I've moved it back up and it seems to work. I included the `html.` so you can change the class name manually in the inspector and not have to toggle js in the settings and reload the page.